lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -20,329 +20,378 @@
 options:
   name:
     required: true
+    type: str
     description:
-    - Service name
+      - Service name
   image:
-    type: str
     required: true
+    type: str
     description:
-    - Service image path and tag.
-      Maps docker service IMAGE parameter.
+      - Service image path and tag.
+      - Maps docker service C(IMAGE) parameter.
```

Comment:
I think it would make more sense to write this (and the later ones) as
```suggestion
      - Maps C(docker service create)'s C(IMAGE) parameter.
```
or
```suggestion
      - Corresponds to the C(IMAGE) parameter of C(docker service create).
```
```
$ docker service create --help

Usage:	docker service create [OPTIONS] IMAGE [COMMAND] [ARG...]
```